portal/src/OnboardingSurveyScreen.tsx

```diff
@@ -391,39 +396,43 @@ function Step1(_props: StepProps) {
 
 function Step2(_props: StepProps) {
   const prefix = "step2";
-  const toriChoiceGroup = "teamOrIndividualChoiceGroup";
-  const toriChoices = ["Team", "Individual"];
-  const toriChoicesFromLocalStorage = getFromLocalStorage("toriChoices");
-  const defaultToriChoices: string[] = (
-    toriChoicesFromLocalStorage === undefined
+  const TorPChoiceGroup = "teamOrPersonalChoiceGroup";
+  const TorPChoices = ["team", "personal"];
+  const TorPChoicesFromLocalStorage = getFromLocalStorage(
```

Comment:
Lets use the prefix `teamOrPersonal` instead of `TorP` which looks like a class.